pypeit/core/coadd.py

```diff
@@ -567,6 +564,31 @@ def solve_poly_ratio(wave, flux, ivar, flux_ref, ivar_ref, norder, mask = None,
     flux_med, ivar_med = median_filt_spec(flux, ivar, mask, med_width)
     flux_ref_med, ivar_ref_med = median_filt_spec(flux_ref, ivar_ref, mask_ref, med_width)
 
+    if 'poly' in model:
```

Comment:
Add a doc string describing this next block

Reply:
Done.